config/locales/en.yml

```diff
@@ -10,6 +10,7 @@ en:
   none: None
   not_found: Not Found
   please_sign_up: Access Denied. Please sign up for an account at https://rubygems.org
+  otp_incorrect_or_missing: You have enabled multifactor authentication but your OTP code is missing or incorrect. Please check it and retry.
```

Comment:
> your OTP code is missing or incorrect.

this is unnecessarily ambiguous. Can we please add a check for otp being empty and show error message accordingly?

Reply:
Right. But change to this may affect rubygems client, I'll check more of it.